### PR TITLE
Add "remote available" option

### PR DESCRIPTION
**Pull request by meowwwls**
**What**:

For jobs where remote is an option, this adds "Remote Available" next to the location for each job posting. 

**Why**:

Before, if remote was an option, the location was not listed, only "remote" was. Some people looking for jobs might not want to apply for a job even if it is remote if the location is in a different country or time zone, and some jobs have remote as an option in addition to offices. 

**How**:

I added a couple of text job posting to make sure in different situations, the right output would render. Then it was just a matter of adding an `if` statement to the job information section in the `job-listing` nunjucks file.

**Checklist**:

- [ ] Documentation N/A
- [x] Ready to be merged
- [ ] Added myself to contributors table N/A

### CODE REVIEW

File: src/_includes/components/job-listing.njk [hunk 1]
```diff
@@ -5,8 +5,8 @@
   </div>
   <header class="c-job-listing__header">
     <time>Posted on {{ params.date | readableDate }}</time>
-    <h2 class="c-job-listing__title"><a href="">{{ params.title }} at {{ params.company }}</a></h2>
-    <p>Type <strong>{{ params.type }}</strong> &bullet; Location <strong>{% if params.remote === true %}Remote{% else %}{{ params.location }}{% endif %}</strong></p>
+    <h2 class="c-job-listing__title"><a href="">{{ params.title }} at {{ params.company }}</a></h2>    
+    <p>Type <strong>{{ params.type }}</strong> &bullet; Location <strong>{{ params.location }}</strong>{% if params.remote === true %} Remote &bullet; <strong>Available</strong>{% endif %}</p>
```
Comment (alexcarpenter) [inline src/_includes/components/job-listing.njk]: What do you think about wrapping the location in a conditional that checks to see if they have any office?

So if the offices param is set to false, we don't output the location?

`{% if params.offices === true %} &bullet; Location <strong>{{ params.location }}</strong>{% endif %}`

Comment (meowwwls) [reply src/_includes/components/job-listing.njk]: I originally had that and removed it for some reason. 

If offices === false, what should it say? Remote Only, or still Remote Available?

I originally had something like this:

```
<p>Type <strong>{{ params.type }}</strong> &bullet; {% if params.offices === false %} <strong>Remote Only</strong> {% else %} Location <strong>{{ params.location }}</strong>{% if params.remote === true %} Remote &bullet; <strong>Available</strong>{% endif %}{% endif %}</p>
```

Comment (alexcarpenter) [reply src/_includes/components/job-listing.njk]: I like that. Lets move forward with that.

**Comment by alexcarpenter** [comment]
Oh, one other thing to note, is that any jobs that are listed as `open` will show up on the site. Might suggest we delete all but one to use as the template.

**Comment by meowwwls** [comment]
OK, done! I just had those as test postings to make sure the location and remote option would render as expected and should not have pushed them to begin with.

**Pull request merged**